### PR TITLE
Presentation content type

### PR DESCRIPTION
**Pull request by met0ry**
- [x] merged with the latest master branch
- [x] resolved all the conflicts, PR could be merged
- [x] new code has new specs
- [x] for new models - new factories were added too
- [x] unit specs are passed
- [x] code is well-formatted, Ruby, CSS, etc
- [x] screenshots of new pages, features are added
- [x] code passed internal team code review

PS: delete your branch in github after merge http://prntscr.com/q80mvq

### CODE REVIEW

**Comment by met0ry** [comment]
presentation
presentaionShow
presentationNew
presentationFail
Це з factories незнаю на даний момент як пофіксити

File: app/views/account/contents/_presentation_columns.html.slim [hunk 1]
```diff
@@ -0,0 +1,6 @@
+tr
+  td File
+  td
+    = link_to(content.file.url) do
+      = "#{File.basename(content.file.url)}"
```
Comment (vikdotdev) [inline app/views/account/contents/_presentation_columns.html.slim]: тут варто вкоротит ім'я якшо воно більше ніж ~20 символів, вася робив щось подібне у своєму аплоадері на Page - потім винести це у хелпер

Comment (demchyna) [reply app/views/account/contents/_presentation_columns.html.slim]: xxxx

Comment (demchyna) [reply app/views/account/contents/_presentation_columns.html.slim]: create instance method content.file_name in model which will return File.basename(content.file.url)

File: spec/factories/contents.rb [hunk 2]
```diff
@@ -12,7 +12,14 @@
       type { 'Gallery' }
     end
 
+    trait :presentation do
+      type { 'Presentation' }
+      #file { Rack::Test::UploadedFile.new(Rails.root + "spec/files/example.pdf", 'example/pdf') }
```
Comment (demchyna) [inline spec/factories/contents.rb]: remove comment

File: spec/models/presentation_spec.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Presentation, type: :model do
```
Comment (igorkasyanchuk) [inline spec/models/presentation_spec.rb]: додай ще контролер тест де ти створюєш презентацію

**Pull request merged**